static/js/subs.js

```diff
@@ -1117,8 +1117,23 @@ $(function () {
         sub_arrow.removeClass('icon-vector-chevron-up');
         sub_arrow.addClass('icon-vector-chevron-down');
     });
+
+    //Set the width of subscriptions status box with respect to size of
+    //middle column.
+    function set_subscription_status_width() {
+        var column_middle_width = $('.column-middle').width();
+        $('#subscriptions-status').width(0.8*column_middle_width);
+    }
+
+    $(window).resize(function () {
+        set_subscription_status_width();
+    });
+
+    set_subscription_status_width();
```

Comment:
I'm not sure how that would be possible having a fixed element and setting its dimensions with respect to a static element without the use of javascript , one possible way would have been to inherit the width of parent element(in css) but again it is dynamic too(middle column, for this to work width of parent should be defined explicitly in css).

Ah, Let me see if I can do something else.